inspirehep/config.py

```diff
@@ -84,6 +84,7 @@
 CACHE_REDIS_URL = "redis://localhost:6379/0"
 CACHE_TYPE = "redis"
 ACCOUNTS_SESSION_REDIS_URL = "redis://localhost:6379/2"
+ACCESS_CACHE = "invenio_cache:current_cache"
```

Comment:
Can you explain in the commit log why you had to do this?

Reply:
Sure, done!